aggregator/src/main/resources/application.properties

```diff
@@ -4,22 +4,34 @@ quarkus.http.port=8086
 %test.quarkus.http.port=9086
 %test.quarkus.http.test-port=9086
 
+## configure your datasource
+#quarkus.datasource.db-kind=postgresql
+#quarkus.datasource.username=postgres
+#quarkus.datasource.password=postgres
+#quarkus.datasource.jdbc.url=jdbc:postgresql://127.0.0.1:5432/notifications
 # configure your datasource
-quarkus.datasource.db-kind=postgresql
+quarkus.datasource.db-kind=postgres
```

Comment:
Why `postgres` here ? Doc says `postgresql` https://quarkus.io/guides/datasource

Reply:
Will change it back to postgresql. The other one worked with my local postgres database.